tests/test_sdss_reconstruct.py

```diff
@@ -32,5 +32,5 @@ def test_sdss_reconstruct(self, cfg, decoder):
         flux_diff = res_bright.abs().sum()
```

Comment:
Have to update this file to get the correct output from `mock_predict` - it should fail for this commit.

Reply:
(This is all set now)